test/validate.js

```diff
@@ -9,17 +9,17 @@ const { ValidationError, MetaschemaError } = require('../lib/schema-errors');
 
 const validateTest = metatests.test('validate');
 
-const metaschemaPath = path.join(__dirname, '..', 'schemas', 'metaschema');
+// const metaschemaPath = path.join(__dirname, '..', 'schemas', 'metaschema');
```

Comment:
I forgot to delete it. We don't have to load schemas from `metaschema` directory, because `Domain.schema` is the one invalid schema and should not be validated